### PR TITLE
UE: Tnc updates

### PR DESCRIPTION
**Pull request by TalmizAhmed**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--aem-boilerplate-forms--adobe-rnd.hlx.live/
- After: https://ue-tnc-updates--aem-boilerplate-forms--adobe-rnd.hlx.live/

### CODE REVIEW

**Comment by vdua** [comment]
I can't see the component in action in the provided url
https://ue-tnc-updates--aem-boilerplate-forms--adobe-rnd.hlx.live/

File: blocks/form/components/toggleable-link/toggleable-link.js [hunk 1]
```diff
@@ -24,6 +24,14 @@ function decorateLink(link) {
   return link;
 }
 
+/**
+ // eslint-disable-next-line max-len
```
Comment (nit23uec) [inline blocks/form/components/toggleable-link/toggleable-link.js]: pls avoid disabling eslint

File: scripts/form-editor-support.js [hunk 2]
```diff
@@ -96,7 +96,7 @@ function annotateFormFragment(fragmentFieldWrapper, fragmentDefinition) {
 }
 
 function getPropertyModel(fd) {
-  if (!fd[':type'] || fd[':type'].startsWith('core/fd/components') || fd[':type'] === 'wizard') {
+  if (!fd[':type'] || fd[':type'].startsWith('core/fd/components') || fd[':type'] === 'wizard' || fd[':type'] === 'toggleable-link') {
```
Comment (nit23uec) [inline scripts/form-editor-support.js]: lets simplify this:
```
const OOTBComponentsWithViewType = ['wizard', 'toggleable-link'];
OOTBComponentsWithViewType.includes(fd[':type'])
```

File: blocks/form/components/tnc/tnc.js [hunk 3]
```diff
@@ -19,6 +19,9 @@ class TermsAndConditions {
 
   decorate() {
     const textWrapper = this.fieldDiv.querySelector('.plain-text-wrapper');
+    if (!textWrapper) { // rendition does not have a plain-text-wrapper => link rendition of TnC
+      return;
```
Comment (nit23uec) [inline blocks/form/components/tnc/tnc.js:24]: lets print a console.debug msg here.

**Comment by nit23uec** [comment]
> I can't see the component in action in the provided url https://ue-tnc-updates--aem-boilerplate-forms--adobe-rnd.hlx.live/

this url is still displaying doc based forms collateral. to have AEM based forms would mean to install collateral on AEM instance in the pipeline and then publishing the form also.

**Comment by TalmizAhmed** [comment]
Had to take to draft to avoid merge to fix model rendition of tnc as it was rendering panel's model. Tested and it works now.

**Review by nit23uec — APPROVED** [review]

**Pull request merged**